lenses_python/WebSocketHandler.py

```diff
@@ -71,7 +71,16 @@ def on_message(self, ws, message):
                 for i in lst:
                     pp(i)
         elif message['type'] == 'ERROR':
```

Comment:
`error` and `invalid request` have exactly the same code. better to use an `or` here.